stable_baselines3/common/monitor.py

```diff
@@ -85,7 +81,7 @@ def reset(self, **kwargs) -> GymObs:
             self.current_reset_info[key] = value
         return self.env.reset(**kwargs)
 
-    def step(self, action: Union[np.ndarray, int]) -> GymStepReturn:
+    def step(self, action: Union[np.ndarray, int]) -> "GymStepReturn":
```

Comment:
why as string? what was causing an error?

Reply:
So when I import the `ResultsWriter` in the `vec_monitor.py`, it causes a circular import.